packages/components/atoms/f-button/src/components/Button.vue

```diff
@@ -8,9 +8,9 @@
             $style[`o-btn--size${buttonSizeClassname}`],
             (isFullWidth ? $style['o-btn--fullWidth'] : '')
         ]"
-        :attributes="$attrs"
         :action-type="buttonActionType"
         :data-test-id="`${componentType}-component`"
+        v-bind="$attrs"
```

Comment:
As the resolved `component` element ends up being the root element of the `Button` component, and we haven't set [`inheritAttrs: false`](https://vuejs.org/v2/guide/components-props.html#Disabling-Attribute-Inheritance) for `Button`, is it necessary to set this?

Reply:
Looks like it isn't, I have removed this now, thanks!